src/builder/edit_thread.rs

```diff
@@ -7,6 +7,7 @@ use crate::internal::prelude::*;
 #[cfg(feature = "http")]
 use crate::model::prelude::*;
```

Comment:
```suggestion
use crate::model::prelude::*;
```
This will fix the CI failures.